common/djangoapps/student/views/dashboard.py

```diff
@@ -572,9 +598,12 @@ def student_dashboard(request):
         'EMPTY_DASHBOARD_MESSAGE', None
     )
 
+    disable_course_limit = request and 'course_limit' in request.GET
```

Comment:
You should update the name of the query param as well, so that it is the same, `course_limit` -> `disable_course_limit`

Reply:
@fenail earlier I was using ''load all courses" parameter for loading all the courses. But @ormbsbee suggested we should only use one variable either it should be course limit or load all courses. That's why I have only used course limit variable.

Reply:
ok, we can go with that for now.

Reply:
@fenail sure I'll do that. One question I have who should I contact begore merging this PR or You will merge this Pr?

Reply:
@adeelehsan I assumed that after approval you'll merge this PR to master as it is your ticket/work.